Add highlights for week ending 2022-04-08 (#1389)
giantswarm/docs: src/content/changes/highlights/2022-04-08.md

---
date: 2022-04-08T10:00:00
title: Highlights for the week ending April 8, 2022
changes_categories:
  - Highlights
owner:
  - https://github.com/orgs/giantswarm/teams/sig-product
---

## Apps

> 

Reviewer: ```suggestion
- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) versions [v2.10.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#2100---2022-04-04) [v2.1.4](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#214---2022-04-07) fix [CVE-2022-0778](https://github.com/kubernetes/ingress-nginx/issues/8339) in OpenSSL and [CVE-2022-23308](https://github.com/kubernetes/ingress-nginx/issues/8321) in libxml2.
```

> 

Reviewer 2: ```suggestion
- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.10.0](https://github.com/giantswarm/kyverno-app/blob/master/CHANGELOG.md#0100---2022-04-05) updates to Kyverno version 1.6.2 including performance and stability improvements.
- [kyverno-policies](https://github.com/giantswarm/kyverno-policies/) version [v0.17.1](https://github.com/giantswarm/kyverno-policies/blob/main/CHANGELOG.md#0171---2022-04-06) includes policies for enforcing Kubernetes Pod Security Standards (PSS). This is the first release of this app intended for use outside Giant Swarm's own clusters.
- [security-pack](https://github.com/giantswarm/security-pack) version [v0.1.0](https://github.com/giantswarm/security-pack/blob/main/CHANGELOG.md#010---2022-04-07) enables Kyverno installation by default, updates to Falco app version 0.3.2, and includes the `kyverno-policies` app for PSS policy enforcement.
```

- [kyverno-app](https://github.com/giantswarm/kyverno-app) version [v0.10.0](https://github.com/giantswarm/kyverno-app/blob/master/CHANGELOG.md#0100---2022-04-05) updates to Kyverno version 1.6.2 including performance and stability improvements.
- [kyverno-policies](https://github.com/giantswarm/kyverno-policies/) version [v0.17.1](https://github.com/giantswarm/kyverno-policies/blob/main/CHANGELOG.md#0171---2022-04-06) includes policies for enforcing Kubernetes Pod Security Standards (PSS). This is the first release of this app intended for use outside Giant Swarm's own clusters.
- [security-pack](https://github.com/giantswarm/security-pack) version [v0.1.0](https://github.com/giantswarm/security-pack/blob/main/CHANGELOG.md#010---2022-04-07) enables Kyverno installation by default, updates to Falco app version 0.3.2, and includes the `kyverno-policies` app for PSS policy enforcement.

> - [efk-stack-app](https://github.com/giantswarm/efk-stack-app) version [v.0.7.3](https://github.com/giantswarm/efk-stack-app/blob/master/CHANGELOG.md#073---2022-04-06) adds team annotations in chart.yaml for alert routing and fixes deprecated api for rbac.

Reviewer: ```suggestion
- [efk-stack-app](https://github.com/giantswarm/efk-stack-app) version [v0.7.3](https://github.com/giantswarm/efk-stack-app/blob/master/CHANGELOG.md#073---2022-04-06) adds team annotations in Chart.yaml for alert routing and fixes deprecated api for rbac.
```

> - [dashboards](https://github.com/giantswarm/dashboards) versions [v.2.1.0](https://github.com/giantswarm/dashboards/blob/master/CHANGELOG.md#210---2022-04-05) and v[2.0.0](https://github.com/giantswarm/dashboards/blob/master/CHANGELOG.md#200---2022-04-04) adds all dashboards form g8s-grafana, grafana sidecar annotation to all config maps, dashboard for ceph cluster usage in KVM. Splits each dashboards in specific configmaps and makes the mixin dashboard private.

Reviewer: ```suggestion
- [dashboards](https://github.com/giantswarm/dashboards) versions [v2.1.0](https://github.com/giantswarm/dashboards/blob/master/CHANGELOG.md#210---2022-04-05) and [v2.0.0](https://github.com/giantswarm/dashboards/blob/master/CHANGELOG.md#200---2022-04-04) adds all dashboards form g8s-grafana, grafana sidecar annotation to all config maps, dashboard for ceph cluster usage in KVM. Splits each dashboards in specific configmaps and makes the mixin dashboard private.
```
